creme/time_series/exponential_smoothing.py

```diff
@@ -1,56 +1,89 @@
+from typing import List, Optional
 import collections  # replace by creme utils windows
 from . import base
 
 
-class _DampedSmoothing:
-    @staticmethod
-    def _discount_sum(phi: float, h: int) -> list:
-        return sum([phi**i for i in range(1, h + 1)])
+def _discount_sum(phi: float, h: int) -> list:
+    return sum([phi**i for i in range(1, h + 1)])
 
 
-class ExponentialSmoothing(base.Forecaster):
+class SimpleExponentialSmoothing(base.Forecaster):
     """
-     #TODO : Docstring + test
+     Simple exponential smoothing.
+     Mathematically, it is defined as:
+     Forecast equation
+     .. math:: \hat{y}_{t+h | t}=\ell_{t}
+     Smoothing equation
+     .. math:: \ell_{t}=\alpha y_{t}+(1-\alpha) \ell_{t-1} 
 
     Parameters:
-        alpha (float): Defaults to `0.5`.
+        alpha (float): The smoothing parameter for the level, 0 ≤ alpha ≤ 1 . Defaults to `0.5`.
+        l0 (float): Initialization value for the level. Defaults to `0`.
+
+    Example:
+        ::
+            >>> #TODO
 
     References:
         1. `Simple exponential smoothing <https://otexts.com/fpp2/ses.html>`_
     """
 
-    def __init__(self, alpha=0.5):
-        self.alpha = alpha
-        self.st = 0
+    def __init__(self, alpha=0.5, l0: float = 0):
+
+        if 0 <= alpha <= 1:
```

Comment:
I would instead do:

```python
if not 0 <= alpha <= 1:
    raise ValueError('blablabla')
self.alpha = alpha
```

That goes as well for the other parameters in the other classes :)